# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fbset/install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fbset-2.1/Makefile.orig	2006-08-01 17:26:37.000000000 +0200
+++ fbset-2.1/Makefile	2006-08-01 17:33:04.000000000 +0200
@@ -24,10 +24,9 @@
 		$(BISON) modes.y
 
 install:	fbset
-		if [ -f /sbin/fbset ]; then rm /sbin/fbset; fi
-		$(INSTALL) fbset /usr/sbin
-		$(INSTALL) fbset.8 /usr/man/man8
-		$(INSTALL) fb.modes.5 /usr/man/man5
+		$(INSTALL) fbset $(DESTDIR)/sbin
+		$(INSTALL) fbset.8 $(DESTDIR)/usr/man/man8
+		$(INSTALL) fb.modes.5 $(DESTDIR)/usr/man/man5
 
 clean:
 		$(RM) *.o fbset lex.yy.c modes.tab.c modes.tab.h
